applications/asset_tracker_v2/src/main.c

```diff
@@ -480,8 +482,10 @@ void main(void)
 		sys_reboot(SYS_REBOOT_COLD);
 	} else {
 		SEND_EVENT(app, APP_EVT_START);
+		module_set_state(MODULE_STATE_READY);
```

Comment:
let's move this new line before earlier send event to make sure this is handled before anything else in the app
due to how scheduling works this should change nothing but it is more logical